README.md

```diff
@@ -66,6 +71,15 @@ You don't have to ever use `eject`. The curated feature set is suitable for smal
 
 
 
+## Author
+
+👤 **Samiullah Bahadur**
+
+- [Github](https://github.com/samiullahbahadur)
+- [LinkedIn](https://www.linkedin.com/in/samiullah-bahadur/)
+
+
+
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md) for this project, " ` The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that you give appropriate credit to the author. Therefore, you should do it in the README of your project. ` ", kindly acknowledge the author of the original design.  In your `Acknowledgements` section, gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

Reply:
okay I will